### PR TITLE
document nested services.

### PR DESCRIPTION
**Pull request by mwhooker**

### CODE REVIEW

File: content/docs/services/advanced-services.md [hunk 1]
```diff
@@ -73,3 +73,43 @@ build:
 ```
 
 Here we define the `mariadb` service to be only available in the [build pipeline](http://devcenter.wercker.com/learn/pipelines/introduction.html).
+
+### Nested services
+
+You can develop your services locally using nested services. The service syntax
+is largely the same, except we add a url field, which points to the service you
+want to use on disk.
```
Comment (Tonnu) [inline content/docs/services/advanced-services.md]: Some more information here would be nice; i.e pointing out that the `#dev` part of the URL points to the `dev clause` of the wercker.yml of that project.

File: content/docs/services/advanced-services.md [hunk 2]
```diff
@@ -73,3 +73,43 @@ build:
 ```
 
 Here we define the `mariadb` service to be only available in the [build pipeline](http://devcenter.wercker.com/learn/pipelines/introduction.html).
+
+### Nested services
+
+You can develop your services locally using nested services. The service syntax
```
Comment (Tonnu) [inline content/docs/services/advanced-services.md]: You can develop your service-oriented applications by combining different wercker projects; we call this 'nesting services'.

File: content/docs/services/advanced-services.md [hunk 3]
```diff
@@ -73,3 +73,43 @@ build:
...
+box: python:2.7
+dev:
+  steps:
+    - pip-install
+    - internal/watch:
+        name: start web server
+        code: python app.py
+        reload: false
+```
+
```
Comment (Tonnu) [inline content/docs/services/advanced-services.md]: maybe paste in the wercker.yml of the api service for clarity?

**Comment by mwhooker** [comment]
from slack:

> toon [9:17 AM] 
:squirrel:

**Pull request merged**